README.md

```diff
@@ -33,27 +32,28 @@ FROM testbed/vim:latest
 
 RUN install_vim -tag v7.3 -name vim73 -build \
                 -tag v7.4.052 -name vim74 -build \
-                -tag master -build
+                -tag master -build \
+                -vim neovim -tag master -py2 -py3 -build
```

Comment:
I will slightly enhance the examples in example/Dockerfile, and use them here.  I've thought about using those in the tests then (automatically), and did it now.. :)